docs/components/controls.md

```diff
@@ -0,0 +1,122 @@
+---
+title: controls
+type: components
+layout: docs
+parent_section: components
+order: 4
+---
+
+The controls component defines the position and rotation of an entity, based on input from one or more input devices. Each input method is attached to its own component, and can be enabled/disabled or configured through that component.
+
+The following input components are provided by A-Frame, in addition to any custom controls created by the user:
+
+* [gamepad-controls](#gamepad-controls) *(position + rotation)*
+* [hmd-controls](#head-mounted-display-controls) *(position + rotation)*
+* [keyboard-controls](#keyboard-controls) *(position)*
+* [mouse-controls](#mouse-controls) *(rotation)*
+* [touch-controls ](#touch-controls) *(position)*
+
+[//]: # (HMD-controls link is spelled out, to prevent Hexo from replacing "md" with ".html".)
```

Comment:
hah yeah, is `data.path` the current page, or the target? Maybe just need to escape the "." in the line 49 regexp.